playbooks/roles/prereqs/tasks/install_virtual_reqs.yml

```diff
@@ -47,5 +38,4 @@
   - jq
   - libselinux-python
   - git
-  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
```

Comment:
I did not quite understand the logic behind dropping fedora. If we only support rpm based, I guess it's better to drop the condition itself, imo.

Reply:
@skatlapa it is for setting up EPEL or not setting up EPEL and before it was for your yum and dnf.
So now there is no need on this line since we will always install these packages using package.